brooklyn-tosca-transformer/src/main/java/io/cloudsoft/tosca/a4c/brooklyn/ToscaTypePlanTransformer.java

```diff
@@ -167,4 +180,28 @@ private void assertAvailable() {
         }
     }
 
+    @Override
+    public double scoreForTypeDefinition(String formatCode, Object catalogData) {
+        return 0; // TODO: Not yet implemented
+    }
+
+    @Override
+    public List<RegisteredType> createFromTypeDefinition(String formatCode, Object catalogData) {
+        throw new UnsupportedOperationException();
```

Comment:
I see `JavaClassNameTypePlanTransformer` throws. May be worth clarifying how this method is used.